reference/tax_provider.yml

```diff
@@ -856,7 +856,10 @@ components:
           description: A unique identifier for this item used to map responses back to the corresponding item on the order.
         item_code:
           type: string
-          description: 'The UPC/SKU of the item. The UPC value will be used when both UPC and SKU values are available on the item.'
+          description: 'The UPC of the item. If UPC is not available, the SKU value will be used. Empty string if both UPC and SKU are not available.'
```

Comment:
This is still UPC/SKU, the change itself is additive

Reply:
Thanks, I edited the description